meta-mender-core/recipes-mender/mender-client/mender-client_git.bb

```diff
@@ -1,4 +1,4 @@
-require mender-client-go.inc
+require mender-client-cpp.inc
```

Comment:
What's the benefit?

Reply:
None! 😄 I messed this up, we should of course be using the `mender_git.bb` recipe instead. I'll remove this commit, thanks for catching.